Tasks/AzureStaticWebAppV0/task.loc.json

```diff
@@ -14,7 +14,7 @@
   "demands": [],
   "version": {
     "Major": 0,
-    "Minor": 243,
+    "Minor": 244,
```

Comment:
I think they would want it as 246 https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md https://whatsprintis.it/

Reply:
task.json needs to be updated as well https://github.com/microsoft/azure-pipelines-tasks/pull/20461/files#diff-89f5d6e13c90cb179acb54fc3d877f3852adba33d6761572d45222e3ae4ed91fR17